src/features/content-answers/ContentAnswers.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react';
 
+import ContentAnswersModal from './ContentAnswersModal';
 import ContentAnswersOpenButton from './ContentAnswersOpenButton';
 
 type ExternalProps = {
```

Comment:
Shouldn't this just be called `Props`?

Reply:
trying to align with how ContentOpenWith does
https://github.com/box/box-ui-elements/blob/09110051c86f29a358c99c44ecba330fffb050dc/src/elements/content-open-with/ContentOpenWith.js#L47-L49